{
	"alias": "FEDORA-2020-7aa962c55e",
	"autokarma": true,
	"autotime": true,
	"bugs": [
		{
			"bug_id": 1870298,
			"feedback": [
				{
					"bug_id": 1870298,
					"comment": {
						"id": 1584009,
						"karma": 1,
						"timestamp": "2020-08-22 06:07:12",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/1a8afd83231ef5ba905344806ba95b39f2abd77b399b0ee217ec3e6fa30a4825?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 1584009
				},
				{
					"bug_id": 1870298,
					"comment": {
						"id": 1584086,
						"karma": 1,
						"timestamp": "2020-08-22 08:29:10",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/137c7cbb63a742835b17d231bd383aaa13a756b3be18150e971ad74198bd96eb?s=24&d=retro",
							"email": "elxreno@gmail.com",
							"groups": [
								{
									"name": "packager"
								}
							],
							"id": 4790,
							"name": "elxreno",
							"openid": "elxreno.id.fedoraproject.org"
						},
						"user_id": 4790
					},
					"comment_id": 1584086
				},
				{
					"bug_id": 1870298,
					"comment": {
						"id": 1584500,
						"karma": 1,
						"timestamp": "2020-08-22 18:30:27",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/b2ae13be7cf78c4fa85d027daac07c34a0ef754116514c10ad0ae13f7d7cbfec?s=24&d=retro",
							"email": "ego.cordatus@gmail.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "rust-sig"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 4120,
							"name": "atim",
							"openid": "atim.id.fedoraproject.org"
						},
						"user_id": 4120
					},
					"comment_id": 1584500
				}
			],
			"parent": true,
			"security": true,
			"title": "CVE-2020-14367 chrony: Insecure writing to PID file",
			"bugzilla": {
				"bug_id": "1870298",
				"alias": "CVE-2020-14367",
				"creation_ts": "2020-08-19 17:25:16 +0000",
				"short_desc": "CVE-2020-14367 chrony: Insecure writing to PID file",
				"delta_ts": "2023-09-26 14:52:33 +0000",
				"bug_status": "CLOSED",
				"resolution": "WONTFIX",
				"keywords": "Security",
				"priority": "medium",
				"bug_severity": "medium",
				"depends_on": [
					"1872720",
					"1872721",
					"1870299",
					"1870312",
					"1870313"
				],
				"blocked": [
					{
						"bug_id": "1866802",
						"error": "NotPermitted"
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "14216329",
						"comment_count": "0",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 17:25:16 +0000",
						"thetext": "When chronyd is configured to save the pidfile in a directory where the chrony user has write permissions (e.g. /var/run/chrony - the default since chrony-3.4), an attacker that compromised the chrony user account could create a symbolic link at the location of the pidfile to make chronyd starting with root privileges follow the symlink and write its process ID to a file for which the chrony user doesn't have write permissions, causing a denial of service, or data loss."
					},
					{
						"isprivate": "0",
						"commentid": "14216332",
						"comment_count": "1",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 17:25:36 +0000",
						"thetext": "Created chrony tracking bugs for this issue:\n\nAffects: fedora-all [bug 1870299]"
					},
					{
						"isprivate": "0",
						"commentid": "14216341",
						"comment_count": "2",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 17:28:25 +0000",
						"thetext": "Acknowledgments:\n\nName: Matthias Gerstner (Suse)"
					},
					{
						"isprivate": "0",
						"commentid": "14216656",
						"comment_count": "5",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 20:14:40 +0000",
						"thetext": "There's an issue on chrony when creating the PID file under /var/run/chrony folder. The file is created during chronyd startup, while still running under as root user, and when it's opened for writing chronyd doesn't check if there's already a symbolic link with the same file name. An attack with privileged access may leverage this issue by creating a symlink with the default pid file name point to any destination file in the system, this may cause data loss and/or deny of service as result of the path traversal."
					},
					{
						"isprivate": "0",
						"commentid": "14219170",
						"comment_count": "6",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-20 14:27:08 +0000",
						"thetext": "Upstream commits for this issue:\n\nhttps://git.tuxfamily.org/chrony/chrony.git/commit/util.c?id=7a4c396bba8f92a3ee8018620983529152050c74\nhttps://git.tuxfamily.org/chrony/chrony.git/commit/main.c?id=e18903a6b56341481a2e08469c0602010bf7bfe3"
					}
				]
			}
		},
		{
			"bug_id": 1870299,
			"feedback": [
				{
					"bug_id": 1870299,
					"comment": {
						"id": 1584009,
						"karma": 1,
						"timestamp": "2020-08-22 06:07:12",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/1a8afd83231ef5ba905344806ba95b39f2abd77b399b0ee217ec3e6fa30a4825?s=24&d=retro",
							"email": "bojan@rexursive.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								}
							],
							"id": 198,
							"name": "bojan",
							"openid": "bojan.id.fedoraproject.org"
						},
						"user_id": 198
					},
					"comment_id": 1584009
				},
				{
					"bug_id": 1870299,
					"comment": {
						"id": 1584086,
						"karma": 1,
						"timestamp": "2020-08-22 08:29:10",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/137c7cbb63a742835b17d231bd383aaa13a756b3be18150e971ad74198bd96eb?s=24&d=retro",
							"email": "elxreno@gmail.com",
							"groups": [
								{
									"name": "packager"
								}
							],
							"id": 4790,
							"name": "elxreno",
							"openid": "elxreno.id.fedoraproject.org"
						},
						"user_id": 4790
					},
					"comment_id": 1584086
				},
				{
					"bug_id": 1870299,
					"comment": {
						"id": 1584500,
						"karma": 1,
						"timestamp": "2020-08-22 18:30:27",
						"update_id": 234487,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/b2ae13be7cf78c4fa85d027daac07c34a0ef754116514c10ad0ae13f7d7cbfec?s=24&d=retro",
							"email": "ego.cordatus@gmail.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "rust-sig"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 4120,
							"name": "atim",
							"openid": "atim.id.fedoraproject.org"
						},
						"user_id": 4120
					},
					"comment_id": 1584500
				}
			],
			"security": true,
			"title": "CVE-2020-14367 chrony: Insecure writing to PID file [fedora-all]",
			"bugzilla": {
				"bug_id": "1870299",
				"creation_ts": "2020-08-19 17:25:24 +0000",
				"short_desc": "CVE-2020-14367 chrony: Insecure writing to PID file [fedora-all]",
				"delta_ts": "2020-08-23 01:03:41 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "1870298",
						"alias": "CVE-2020-14367",
						"creation_ts": "2020-08-19 17:25:16 +0000",
						"short_desc": "CVE-2020-14367 chrony: Insecure writing to PID file",
						"delta_ts": "2023-09-26 14:52:33 +0000",
						"bug_status": "CLOSED",
						"resolution": "WONTFIX",
						"keywords": "Security",
						"priority": "medium",
						"bug_severity": "medium",
						"depends_on": [
							"1872720",
							"1872721",
							"1870299",
							"1870312",
							"1870313"
						],
						"blocked": [
							{
								"bug_id": "1866802",
								"error": "NotPermitted"
							}
						],
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "14216329",
								"comment_count": "0",
								"who": {
									"text": "mbenatto",
									"name": "Marco Benatto"
								},
								"bug_when": "2020-08-19 17:25:16 +0000",
								"thetext": "When chronyd is configured to save the pidfile in a directory where the chrony user has write permissions (e.g. /var/run/chrony - the default since chrony-3.4), an attacker that compromised the chrony user account could create a symbolic link at the location of the pidfile to make chronyd starting with root privileges follow the symlink and write its process ID to a file for which the chrony user doesn't have write permissions, causing a denial of service, or data loss."
							},
							{
								"isprivate": "0",
								"commentid": "14216332",
								"comment_count": "1",
								"who": {
									"text": "mbenatto",
									"name": "Marco Benatto"
								},
								"bug_when": "2020-08-19 17:25:36 +0000",
								"thetext": "Created chrony tracking bugs for this issue:\n\nAffects: fedora-all [bug 1870299]"
							},
							{
								"isprivate": "0",
								"commentid": "14216341",
								"comment_count": "2",
								"who": {
									"text": "mbenatto",
									"name": "Marco Benatto"
								},
								"bug_when": "2020-08-19 17:28:25 +0000",
								"thetext": "Acknowledgments:\n\nName: Matthias Gerstner (Suse)"
							},
							{
								"isprivate": "0",
								"commentid": "14216656",
								"comment_count": "5",
								"who": {
									"text": "mbenatto",
									"name": "Marco Benatto"
								},
								"bug_when": "2020-08-19 20:14:40 +0000",
								"thetext": "There's an issue on chrony when creating the PID file under /var/run/chrony folder. The file is created during chronyd startup, while still running under as root user, and when it's opened for writing chronyd doesn't check if there's already a symbolic link with the same file name. An attack with privileged access may leverage this issue by creating a symlink with the default pid file name point to any destination file in the system, this may cause data loss and/or deny of service as result of the path traversal."
							},
							{
								"isprivate": "0",
								"commentid": "14219170",
								"comment_count": "6",
								"who": {
									"text": "mbenatto",
									"name": "Marco Benatto"
								},
								"bug_when": "2020-08-20 14:27:08 +0000",
								"thetext": "Upstream commits for this issue:\n\nhttps://git.tuxfamily.org/chrony/chrony.git/commit/util.c?id=7a4c396bba8f92a3ee8018620983529152050c74\nhttps://git.tuxfamily.org/chrony/chrony.git/commit/main.c?id=e18903a6b56341481a2e08469c0602010bf7bfe3"
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "14216330",
						"comment_count": "0",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 17:25:24 +0000",
						"thetext": "\nThis is an automatically created tracking bug!  It was created to ensure\nthat one or more security vulnerabilities are fixed in affected versions\nof fedora-all.\n\nFor comments that are specific to the vulnerability please use bugs filed\nagainst the \"Security Response\" product referenced in the \"Blocks\" field.\n\nFor more information see:\nhttp://fedoraproject.org/wiki/Security/TrackingBugs\n\nWhen submitting as an update, use the fedpkg template provided in the next\ncomment(s).  This will include the bug IDs of this tracking bug as well as\nthe relevant top-level CVE bugs.\n\nPlease also mention the CVE IDs being fixed in the RPM changelog and the\nfedpkg commit message.\n\nNOTE: this issue affects multiple supported versions of Fedora. While only\none tracking bug has been filed, please correct all affected versions at\nthe same time.  If you need to fix the versions independent of each other,\nyou may clone this bug as appropriate."
					},
					{
						"isprivate": "0",
						"commentid": "14216331",
						"comment_count": "1",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2020-08-19 17:25:27 +0000",
						"thetext": "Use the following template to for the 'fedpkg update' request to submit an\nupdate for this issue as it contains the top-level parent bug(s) as well as\nthis tracking bug.  This will ensure that all associated bugs get updated\nwhen new packages are pushed to stable.\n\n=====\n\n# bugfix, security, enhancement, newpackage (required)\ntype=security\n\n# low, medium, high, urgent (required)\nseverity=medium\n\n# testing, stable\nrequest=testing\n\n# Bug numbers: 1234,9876\nbugs=1870298,1870299\n\n# Description of your update\nnotes=Security fix for [PUT CVEs HERE]\n\n# Enable request automation based on the stable/unstable karma thresholds\nautokarma=True\nstable_karma=3\nunstable_karma=-3\n\n# Automatically close bugs when this marked as stable\nclose_bugs=True\n\n# Suggest that users restart after update\nsuggest_reboot=False\n\n======\n\nAdditionally, you may opt to use the bodhi web interface to submit updates:\n\nhttps://bodhi.fedoraproject.org/updates/new"
					},
					{
						"isprivate": "0",
						"commentid": "14217683",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2020-08-20 07:42:26 +0000",
						"thetext": "FEDORA-2020-7aa962c55e has been submitted as an update to Fedora 32. https://bodhi.fedoraproject.org/updates/FEDORA-2020-7aa962c55e"
					},
					{
						"isprivate": "0",
						"commentid": "14220822",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2020-08-21 01:17:40 +0000",
						"thetext": "FEDORA-2020-7aa962c55e has been pushed to the Fedora 32 testing repository.\nIn short time you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --advisory=FEDORA-2020-7aa962c55e`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2020-7aa962c55e\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "14224779",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2020-08-23 01:03:41 +0000",
						"thetext": "FEDORA-2020-7aa962c55e has been pushed to the Fedora 32 stable repository.\nIf problem still persists, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "chrony-3.5.1-1.fc32",
			"release_id": 35,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "aarch64"
					},
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "aarch64"
					},
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "aarch64"
					}
				],
				"armv7hl": [
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "armv7hl"
					},
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "armv7hl"
					},
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "armv7hl"
					}
				],
				"i686": [
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "i686"
					},
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "i686"
					},
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "i686"
					}
				],
				"ppc64le": [
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "ppc64le"
					},
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "ppc64le"
					},
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "s390x"
					},
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "s390x"
					},
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "chrony-debugsource",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "x86_64"
					},
					{
						"name": "chrony",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "x86_64"
					},
					{
						"name": "chrony-debuginfo",
						"version": "3.5.1",
						"release": "1.fc32",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 1579950,
			"text": "This update has been submitted for testing by mlichvar. ",
			"timestamp": "2020-08-20 07:42:22",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 1579951,
			"text": "This update's test gating status has been changed to 'ignored'.",
			"timestamp": "2020-08-20 07:42:22",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 1579952,
			"text": "This update's test gating status has been changed to 'waiting'.",
			"timestamp": "2020-08-20 07:42:23",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 1580013,
			"text": "This update's test gating status has been changed to 'ignored'.",
			"timestamp": "2020-08-20 08:09:49",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 1581589,
			"text": "This update has been pushed to testing.",
			"timestamp": "2020-08-21 01:18:02",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 1870298,
						"parent": true,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file"
					},
					"bug_id": 1870298,
					"comment_id": 1584009
				},
				{
					"bug": {
						"bug_id": 1870299,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file [fedora-all]"
					},
					"bug_id": 1870299,
					"comment_id": 1584009
				}
			],
			"id": 1584009,
			"karma": 1,
			"timestamp": "2020-08-22 06:07:12",
			"update_id": 234487,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/1a8afd83231ef5ba905344806ba95b39f2abd77b399b0ee217ec3e6fa30a4825?s=24&d=retro",
				"email": "bojan@rexursive.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					}
				],
				"id": 198,
				"name": "bojan",
				"openid": "bojan.id.fedoraproject.org"
			},
			"user_id": 198
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 1870298,
						"parent": true,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file"
					},
					"bug_id": 1870298,
					"comment_id": 1584086
				},
				{
					"bug": {
						"bug_id": 1870299,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file [fedora-all]"
					},
					"bug_id": 1870299,
					"comment_id": 1584086
				}
			],
			"id": 1584086,
			"karma": 1,
			"timestamp": "2020-08-22 08:29:10",
			"update_id": 234487,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/137c7cbb63a742835b17d231bd383aaa13a756b3be18150e971ad74198bd96eb?s=24&d=retro",
				"email": "elxreno@gmail.com",
				"groups": [
					{
						"name": "packager"
					}
				],
				"id": 4790,
				"name": "elxreno",
				"openid": "elxreno.id.fedoraproject.org"
			},
			"user_id": 4790
		},
		{
			"id": 1584087,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2020-08-22 08:30:32",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 1870298,
						"parent": true,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file"
					},
					"bug_id": 1870298,
					"comment_id": 1584500
				},
				{
					"bug": {
						"bug_id": 1870299,
						"security": true,
						"title": "CVE-2020-14367 chrony: Insecure writing to PID file [fedora-all]"
					},
					"bug_id": 1870299,
					"comment_id": 1584500
				}
			],
			"id": 1584500,
			"karma": 1,
			"timestamp": "2020-08-22 18:30:27",
			"update_id": 234487,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/b2ae13be7cf78c4fa85d027daac07c34a0ef754116514c10ad0ae13f7d7cbfec?s=24&d=retro",
				"email": "ego.cordatus@gmail.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "rust-sig"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 4120,
				"name": "atim",
				"openid": "atim.id.fedoraproject.org"
			},
			"user_id": 4120
		},
		{
			"id": 1584501,
			"text": "This update has been submitted for stable by bodhi. ",
			"timestamp": "2020-08-22 18:30:28",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 1584803,
			"text": "This update has been pushed to stable.",
			"timestamp": "2020-08-23 01:03:48",
			"update_id": 234487,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"critpath": true,
	"date_pushed": "2020-08-23 01:03:15",
	"date_stable": "2020-08-23 01:03:15",
	"date_submitted": "2020-08-20 07:42:22",
	"date_testing": "2020-08-21 01:17:29",
	"karma": 3,
	"meets_testing_requirements": true,
	"notes": "Security fix for CVE-2020-14367",
	"pushed": true,
	"release": {
		"branch": "f32",
		"candidate_tag": "f32-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f32",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 32",
		"mail_template": "fedora_errata_template",
		"name": "F32",
		"override_tag": "f32-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f32-signing-pending",
		"pending_stable_tag": "f32-updates-pending",
		"pending_testing_tag": "f32-updates-testing-pending",
		"stable_tag": "f32-updates",
		"state": "archived",
		"testing_repository": "updates-testing",
		"testing_tag": "f32-updates-testing",
		"version": "32"
	},
	"require_bugs": true,
	"require_testcases": true,
	"severity": "medium",
	"stable_days": 14,
	"stable_karma": 3,
	"status": "stable",
	"suggest": "unspecified",
	"test_gating_status": "ignored",
	"title": "chrony-3.5.1-1.fc32",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2020-7aa962c55e",
	"unstable_karma": -3,
	"updateid": "FEDORA-2020-7aa962c55e",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/b0dce3af72be4d5491d36e2ec2a96e9c06c36f7f34a0223e26038b54b59eb8b4?s=24&d=retro",
		"email": "mlichvar@redhat.com",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			},
			{
				"name": "cla_redhat"
			},
			{
				"name": "gitnewt"
			},
			{
				"name": "gitmailcap"
			},
			{
				"name": "gitrpmreaper"
			},
			{
				"name": "trust admins"
			}
		],
		"id": 826,
		"name": "mlichvar",
		"openid": "mlichvar.id.fedoraproject.org"
	},
	"version_hash": "ce826fcae069479cb44057cb323d26ee7fd3d14c"
}
